[UI] Resize sidebar on drag event

flavioislima opened the pull request:
Changed the way we resize the sidebar to use the mouse instead of the collapse button.

https://user-images.githubusercontent.com/26871415/212356230-424d647e-3275-4dff-b7c0-a9129b532970.mov

---

Use the following Checklist if you have changed something on the Backend or Frontend:

- [x] Tested the feature and it's working on a current and clean install.
- [ ] Tested the main App features and they are still working on a current and clean install. (Login, Install, Play, Uninstall, Move games, etc.)
- [ ] Created / Updated Tests (If necessary)
- [ ] Created / Updated documentation (If necessary)

arielj commented on src/frontend/components/UI/Sidebar/components/CurrentDownload/index.tsx:
```diff
@@ -54,37 +54,34 @@ export default React.memo(function CurrentDownload({ appName, runner }: Props) {
-                  progress.percent || 0
-                )}%`}</Typography>
-              </Box>
+        <span className="statusIcon" title={`${getStatus()} - ${gameTitle}`}>
```
I'm removing the conditionals from here to show/hide via CSS so it doesn't need rerendering

arielj commented on src/frontend/screens/Login/components/SIDLogin/index.css:
```diff
@@ -1,17 +1,13 @@
   display: grid;
   place-items: center;
-  width: var(--content-width);
+  width: 70vw;
```
I'm not sure about this value

arielj commented on src/frontend/components/UI/Sidebar/index.tsx:
```diff
@@ -34,8 +30,65 @@ export default React.memo(function Sidebar() {
+
   return (
-    <aside className={classNames('Sidebar', { collapsed: sidebarCollapsed })}>
+    <aside ref={sidebarEl} className="Sidebar">
```
`collapsed` class is added with javascript directly instead of using react for it to avoid rerenders

flavioislima commented:
I also changed the QUIT button to always show and added a dialog to config quitting or not.
#2123 and #1571 

![image](https://user-images.githubusercontent.com/26871415/212355714-5f91ee35-1809-4593-b957-e8b0999f9646.png)

arielj reviewed (approved):
looks good to me :+1:
